README.md

```diff
@@ -70,7 +71,30 @@ Before beginning a sprint:
 
 During a sprint we track progress using a [scrum or kanban-like board](#notes)<sup>4</sup> with status columns for To Do, In Progress, Blocked, Ready for Review, Ready for QA, and Done. 
 
-After finishing a sprint we hold a retrospective to discuss what went well, what didn't go so well, and what we decide to do differently next sprint. Part of retrospective involves examining our [burndown chart](https://en.wikipedia.org/wiki/Burn_down_chart) to gauge the accuracy of our estimates and inform how we estimate and commit to the next sprint's tasks. Retrospective is also the perfect time to adjust the process to better suit our needs. 
+After finishing a sprint we hold a retrospective to discuss what went well, what didn't go so well, and what we decide to do differently next sprint. Part of retrospective involves examining our [burndown chart](https://en.wikipedia.org/wiki/Burn_down_chart) to gauge the accuracy of our estimates and inform how we estimate and commit to the next sprint's tasks. Retrospective is also the perfect time to adjust the process to better suit our needs.
+
+## Velocity Tracking
+
+Velocity is a number that represents the amount of work completed by a team during a specific time period. Typically, we talk about “sprint velocity” and “average sprint velocity”. Sprint velocity describes the amount of work completed in a single sprint. Average sprint velocity is an average of the previous *n* sprints (where *n* is something like 2 - 4 sprints). 
+
+> **sprint-velocity** = *total number of points completed* / *total person-hours*
+
+> **average-sprint-velocity** = average **sprint-velocity** of previous *n* sprints
+
+
+The latter is useful in planning the amount of work to commit to in an upcoming sprint. 
+
+Before a sprint is started the team assigns a point value to each task. This point value represents the estimated amount of effort that it will take to complete the task. At the end of the sprint the team uses these point values to calculate the sprint velocity. First, the team sums the point values of the completed tasks to get the *total number of points completed* (Footnote: Only include the point values of tasks that are 100% “done”). Next, the team sums how many hours each team member was able to contribute to the sprint to get *total person-hours*. Finally, the team divides the *total number of points completed* by the number of *total person-hours* in the sprint. The resulting value is the **sprint-velocity**. Dividing the total points by the total hours results in the smallest possible unit of one point, per one person, per one hour. Since **sprint-velocity** is broken down to the smallest unit, it can scale well with any number of team members as well as any number of days in a sprint.
+
```

Comment:
I think providing this detail but keepin it brief while alluding to other parts of the process as you did is awesome. 

Is there a way we could more formally tie how velocity tracking fits into the 5 step process (that corresponds to the diagram) above? (maybe linking to this section would help with that)

One thing I remember that was confusing to me when trying to understand velocity tracking was that it's part of how we both evaluate the prior sprint and plan for the next one. So it's like a chicken and the egg problem when you're trying to explain it's value to people who are unfamiliar. 

Just a thought what if we broke this section down a bit more into 2 small sub sections
- sprint velocity as a concept, any relevant definitions like person hours
- sprint velocity in practice (you can aggregate historical ones, how it helps us plan)